contents/handbook/small-teams/website-docs/index.mdx

```diff
@@ -57,11 +57,33 @@ Some of the design tools we use, like Balsamiq and Figma, both have built-in com
 
 **Important:** We prioritize feedback based on alignment with business goals. _Everyone has feedback about design._ (If feedback is more of a personal opinion than a business-related perspective, we’ll note it, but don't be offended if your feedback isn't specifically addressed!)
 
+## Working with us
+
+If you're in a team that isn't Website & Docs, and you want to make a change to the website, here is a guide to working with us.
+
+Most helpful - objective changes:
+
+* Fixing errors
+* Adding missing functionality
+* Correcting typos
+* Fixing broken linsk
+
+Less helpful / can be net negative - subjective changes:
+
+* CTA changes / additions
+* Wording changes, especially titles on the most prominent pages
+* Menu / navigation structure changes
+* Aesthetic changes
+* Deleting prominent pages / large blocks of content
+
+In general, with the above, it is more than likely that the team has put considerable thought into them and this all will need explaining to you. In other situations, the above are known to be MVP state thus we are working on completely overhauling something - hence discussing and merging a potential iterative improvement may be as much effort as redoing something entirely.
+
+Please do _not_ self merge your own PRs to posthog.com unless it is an emergency. This helps reduce errors.
```

Comment:
I assumed you could self merge once someone else _approves_ your PR, but that you shouldn't just merge your own stuff without approval. Is that ok, or have I been committing a horrendous faux pas here?

Reply:
Is this an absolute rule, or a common sense rule?

I've merged in some of my own PRs (for example when reverting a previous PR, or when it's just a typo that's languished without approval) and now unsure if this is good stepping on toes or bad stepping on toes. I'm fine with either, but want to hit the right balance of speed vs. process.

Reply:
> I assumed you could self merge once someone else approves your PR

This is fine. May be worth a reminder in the next sprint planning that this therefore means if you approve something, you're stamping a "my final vote in it is that this works / is good to go into production" versus "i generally agree with this"


> Is this an absolute rule, or a common sense rule?

being _slightly_ more pedantic than we are today I think would be an improvement, so i'm proposing here

typos - fine

taste-based wording / positioning changes / CTAs - not fine to self-merge. i think we're trying to tightly control this stuff so we have consistency etc

reverting a PR could be either a big change / breaking change or totally fine, depends what it affects. be very mindful of the build not failing or it undoing something intentional. Broadly, i would _not_ do this unless an emergency (ie we have a wild pricing mistake / pr catastrophe unfolding / whatever else)